### PR TITLE
fix: some EN troubleshooting clarifications

### PR DESCRIPTION
**Pull request by Strati**
just the beginning of some troubleshooting-oriented text edits, if this passes review

### CODE REVIEW

**Comment by Strati** [comment]
don't say a word about my failure to squash commits, @ayyaruq :)

**Comment by ayyaruq** [comment]
Strati I love you but have you met our lord and saviour `git rebase`?

File: apps/client/src/assets/i18n/en.json [hunk 1]
```diff
@@ -1661,7 +1661,7 @@
     "Title": "Blog"
   },
   "ERRORS": {
-    "Http_request_error": "There is an issue with {{tool}}, please check #announcements discord channel or report it in the #bugs channel",
+    "Http_request_error": "There is an issue with {{tool}}, please wait a moment and try again. If the issue persists, check the Teamcraft Discord server for updates.",
```
Comment (ayyaruq) [inline apps/client/src/assets/i18n/en.json]: This should probably still reference #announcements.

File: apps/client/src/assets/i18n/en.json [hunk 2]
```diff
@@ -587,9 +587,9 @@
     "Amount_craftable": "You can craft {{amount}}",
     "Add_item": "Add an item",
     "List_large": "This is a large list",
-    "Enable_performance_mode": "You might want to enable performance mode in settings in order to get faster rendering",
+    "Enable_performance_mode": "You might want to enable performance mode in settings in order to get faster rendering. Do not report performance issues displaying this error as a bug, it will not be supported.",
     "List_too_large": "List too large",
-    "List_too_large_description": "This list is too large and might hit database size limitations because of all the informations it carries. You aren't allowed to add more items to make sure you won't lose the list.",
+    "List_too_large_description": "This list is too large and might hit database size limitations because of all the information it carries. You aren't allowed to add more items to make sure you won't lose the list.",
```
Comment (ayyaruq) [inline apps/client/src/assets/i18n/en.json]: The last sentence here kinda sounds like you're saying the user isn't allowed to lose the list. Maybe simplify like this:
> This list is too large and might slow down your computer or hit database limitations. To ensure you won't lose the list, you are no longer able to add further items to it unless existing items are removed.

**Comment by Strati** [comment]
@ayyaruq's suggestions implemented.

**Pull request merged**